dotcom-rendering/src/components/ArticlePage.tsx

```diff
@@ -45,19 +45,15 @@ interface AppProps extends BaseProps {
  * @description
  * returns global styles for article pages
  */
-const globalStyles = (
-	format: ArticleFormat,
-	darkModeInApps: boolean,
-	renderingTarget: RenderingTarget,
-) => css`
+const globalStyles = (format: ArticleFormat, darkMode: boolean) => css`
 	:root {
 		/* Light palette is default on all platforms */
 		${paletteDeclarations(format, 'light')}
 		body {
 			color: ${sourcePalette.neutral[7]};
 		}
-		/* Dark palette only for apps and only if switch turned on */
-		${darkModeInApps && renderingTarget === 'Apps'
+		/* Dark palette only if supported */
+		${darkMode
```

Comment:
Better to keep consistent and use `darkModeAvailable` everywhere?

Reply:
Not sure I understand…?

Reply:
yeah `darkModeAvailable` is used down here https://github.com/guardian/dotcom-rendering/pull/11341/files#diff-a11eccfa997d0d1e670d52c7a3fefeeb296a34ec6411b5e8870aa66b56e69dccR54 seems more semantic?